modules/eks/submodules/privatelink/vpc-endpoint-services.tf

```diff
@@ -17,13 +17,31 @@ data "aws_route53_zone" "hosted" {
   private_zone = false
 }
 
+resource "aws_security_group" "nlb_sg" {
+  name        = "${var.deploy_id}-nlb-sg"
+  description = "NLB Security Group"
+  vpc_id      = var.network_info.vpc_id
+
+  egress {
+    description = "All traffic"
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
could we restrict this?

Reply:
I could use data resource to get CIDR from vpc_id and restrict it to VPC, do you think that makes sense?

Reply:
the infra module should have this information, you can pass it through